### PR TITLE
Add the Conditional Unique for checking speed

### PR DESCRIPTION
**Pull request by PLynx01**
As I said on Discord, I've posted a pull request adding the conditional unique for checking the game speed, alongside with proper parameter type. This unique is simple and straightforward, but I don't know whether the wording is correct.

### CODE REVIEW

File: core/src/com/unciv/models/ruleset/unique/UniqueType.kt [hunk 1]
```diff
@@ -634,6 +634,8 @@ enum class UniqueType(
     ConditionalStartingFromEra("starting from the [era]", UniqueTarget.Conditional),
     ConditionalIfStartingInEra("if starting in the [era]", UniqueTarget.Conditional),
 
+    ConditionalSpeed("when speed is [speed]", UniqueTarget.Conditional),
+
```
Comment (yairm210) [inline core/src/com/unciv/models/ruleset/unique/UniqueType.kt:638]: When *game* speed is []

**Comment by yairm210** [comment]
All in all looks good!
Compact and combinable, just what we want from uniques

**Comment by SeventhM** [comment]
I swear something feels off about it in terms of wording, but that's just me. ~~Would a native speaking real write the speed as a noun like that? Something like "for [speed] game speed" sounds a bit robotic I guess, but Idk feels a bit better. Eh, probably just nitpicking~~

**Comment by yairm210** [comment]
If you can think of a different phrasing, we can do that :)
If not, I'm in favor of merging as-is - we can always change later, and this is a classic case of autoupdate to the rescue - modders will only need to click the button

**Comment by SeventhM** [comment]
By default I'm thinking something like `for the [speed] game speed` or `on [speed] game speed` or at the very very least `when the game speed is [speed]`

**Comment by yairm210** [comment]
That last one was my suggestion :)
"on [] game speed" sounds better to me though, nice! 👍🏿

**Comment by SeventhM** [comment]
> That last one was my suggestion 

Key thing was missing the word "the"

**Comment by SeventhM** [comment]
Mergeable... Translation?

**Comment by yairm210** [comment]
Incorrectly marked, will change the text

**Pull request merged**